packages/portal-proto/src/utils/index.ts

```diff
@@ -289,3 +289,18 @@ export const XL_BREAKPOINT = parseInt(tailwindConfig.theme.extend.screens.xl);
 export const REPO_BREAKPOINT = parseInt(
   tailwindConfig.theme.extend.screens["Custom-Repo-Width"],
 );
+
+// https://stackoverflow.com/questions/2388115/get-locale-short-date-format-using-javascript
+/**
+ * Returns the current local time as a formatted timestamp string.
+ *
+ * @returns A timestamp in the format 'YYYY-MM-DD.HHMMSS'.
+ *
+ * Uses the Swedish ("sv-SE") locale to get a base format of "YYYY-MM-DD HH:MM:SS".
+ */
+export function getFormattedTimestamp() {
```

Comment:
qqn -- have we been using UTC date all over (instead of local date)?

Reply:
We were just using YYYY-MM-DD before but it was UTC I believe.
But because it was just YYYY-MM-DD it didn't matter much IMO. 
Now we are explicitly using local timezone as we require HHMMSS for manifest downloads.

Reply:
@amlehman I update the current file to just use one.

Reply:
Looks great 👍